test/e2e-test/rollout_plan_test.go

```diff
@@ -351,7 +351,7 @@ var _ = Describe("Cloneset based rollout tests", func() {
 				k8sClient.Get(ctx, client.ObjectKey{Namespace: namespace, Name: newAppRollout.Name}, &appRollout)
 				return appRollout.Status.RollingState
 			},
-			time.Second*10, time.Millisecond*50).Should(BeEquivalentTo(oamstd.RollingInBatchesState))
+			time.Second*10, time.Millisecond).Should(BeEquivalentTo(oamstd.RollingInBatchesState))
```

Comment:
same do you mean second?

Reply:
no, I meant to check faster but the same e2e failed anyway, need to see why